scripts/registration/RegisterPluginAndStartStreamDeck.ps1

```diff
@@ -4,13 +4,13 @@ Write-Host "Script root: $PSScriptRoot`n"
 
 $basePath = $PSScriptRoot
 
-if (!($PSSCriptRoot)) {
+if ($PSSCriptRoot.Length -eq 0) {
   $basePath = $PWD.Path;
 }
 
 
 # Load and parse the plugin project file
-$pluginProjectFile = Join-Path $basePath "SamplePlugin.csproj"
+$pluginProjectFile = "$basePath\_StreamDeckPlugin_.csproj"
```

Comment:
This is going to be a problem. Is there a better way to do this that will work in both contexts?